build.sh

```diff
@@ -703,7 +703,18 @@ case "$COMMAND" in
         ;;
 
     "test-osx-object-server")
-        xc "-scheme 'Object Server Tests' -configuration $CONFIGURATION -sdk macosx test"
+        xc "-scheme 'Object Server Tests' -configuration $CONFIGURATION -sdk macosx test \
+            -skip-testing:ObjectServerTests/RLMObjectServerTests/testUsernamePasswordAuthenticationWithReconnection \
+            -skip-testing:ObjectServerTests/RLMObjectServerTests/testAddObjectsWithReconnection \
+            -skip-testing:ObjectServerTests/RLMObjectServerTests/testDownloadRealmWithReconnection"
```

Comment:
What's the purpose of these flags? Why do we want to skip these tests only when testing via `build.sh`?